boards/nordic/thingy53/thingy53_nrf5340_common.dtsi

```diff
@@ -88,7 +88,7 @@
 		regulator-boot-on;
 	};
 
-	vbatt {
+	vbatt: vbatt {
```

Comment:
Is this node label actually needed?

Reply:
Not strictly needed, but its hard to get the device pointer without it